AutomationSelenium/Process/LanguageProcess.cs

```diff
@@ -0,0 +1,175 @@
+﻿using AutomationSelenium.Pages.Components.ProfilePageOverview;
+using AutomationSelenium.Test_Model;
+using AutomationSelenium.Utilities;
+using AventStack.ExtentReports;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AutomationSelenium.Process
+{
+    public class LanguageProcess : Base
+    {
+#pragma warning disable
+        AddEditLanguageComponent addEditLanguageComponentObj;
+        ExtentTest testreport;
+        private ExtentReports extent;
+
+        public LanguageProcess()
+        {
+            addEditLanguageComponentObj = new AddEditLanguageComponent();
+        }
+        public void addedLanguageDetails()
+        {
+           
+            // Read test data from the JSON file using JsonHelper
+            string sFile = "LanguageAddData.json";
+            List<LanguageTestModel> LanguageAddData = Jsonhelper.ReadTestDataFromJson<LanguageTestModel>(sFile);
+            foreach (var data in LanguageAddData)
+            {
+                LogScreenshot("AddLanguage");
+
+                string language = data.language;
+                string level = data.level;
+                addEditLanguageComponentObj.addNewLanguage(data);
+                string addedNewLanguage = addEditLanguageComponentObj.verifyAddedLanguage();
+                Assert.AreEqual(addedNewLanguage, language, "Actual language and expected language do not match");
```

Comment:
you can move all your asserts into helpers like you have done with the description asserts, its will make your code look much cleaner please do this for all asserts